diff --git a/node_modules/@react-navigation/routers/lib/module/index.js b/node_modules/@react-navigation/routers/lib/module/index.js
index 64fcfd6..e762315 100644
--- a/node_modules/@react-navigation/routers/lib/module/index.js
+++ b/node_modules/@react-navigation/routers/lib/module/index.js
@@ -1,8 +1,8 @@
 import * as CommonActions from './CommonActions';
 export { CommonActions };
 export { default as BaseRouter } from './BaseRouter';
-export { default as StackRouter, StackActions, StackActionHelpers, StackActionType, StackRouterOptions, StackNavigationState } from './StackRouter';
-export { default as TabRouter, TabActions, TabActionHelpers, TabActionType, TabRouterOptions, TabNavigationState } from './TabRouter';
-export { default as DrawerRouter, DrawerActions, DrawerActionHelpers, DrawerActionType, DrawerRouterOptions, DrawerNavigationState } from './DrawerRouter';
+export { default as StackRouter, StackActions } from './StackRouter';
+export { default as TabRouter, TabActions } from './TabRouter';
+export { default as DrawerRouter, DrawerActions } from './DrawerRouter';
 export * from './types';
 //# sourceMappingURL=index.js.map
